src/Stratis.Features.FederatedPeg/Models/BuildMultisigTransactionRequest.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using Stratis.Bitcoin.Features.Wallet.Models;
+
+namespace Stratis.Features.FederatedPeg.Models
+{
+    /// <summary>
+    /// A class containing the necessary parameters for a build transaction request.
+    /// </summary>
+    public class BuildMultisigTransactionRequest
+    {
+        /// <summary>
+        /// A list of transaction recipients. For each recipient, specify the Pubkey script and the amount the
```

Comment:
The recipient isn't really a pubkey, it's just any address considered valid by the current network. Minor nitpick

Reply:
I copy-pasted this from the main wallet 😅